nav-admin.php

```diff
@@ -0,0 +1,5 @@
+<?php  if(isset($_SESSION['admin_status']) && $_SESSION['admin_status'] == 1) : ?>
```

Comment:
Hey, here's an advice.

Instead of wrapping your whole function/template/code in a IF you can directly short circuit it.
```suggestion
<?php
if ( ! isset( $_SESSION['admin_status'] ) ||  1 !== $_SESSION['admin_status'] ) {
    return;
}
```

Reply:
fixed